data/pigui/modules/ship-internals-window.lua

```diff
@@ -131,25 +96,34 @@ local function displayShipFunctionWindow()
 	if ui.optionsWindow.isOpen then return end
 	player = Game.player
 	local current_view = Game.CurrentView()
-	local buttons = 6
-	ui.setNextWindowPos(Vector2(ui.screenWidth/2 - ui.reticuleCircleRadius - (mainButtonSize.x + 2 * mainButtonFramePadding) * buttons, ui.screenHeight - mainButtonSize.y * 3 - 8) , "Always")
-	ui.window("ShipFunctions", {"NoTitleBar", "NoResize", "NoFocusOnAppearing", "NoBringToFrontOnFocus", "NoSavedSettings"},
+	local buttons = 3
+	local thrust_widget_size = Vector2(mainButtonSize.x * 3, mainButtonSize.y * 2)
+	assert(thrust_widget_size.y >= mainButtonSize.y)
+	local window_width = ui.getWindowPadding().x * 2 + (mainButtonSize.x + 2 * mainButtonFramePadding + ui.getItemSpacing().x) * buttons + thrust_widget_size.x + mainButtonFramePadding * 2
+	local window_height = thrust_widget_size.y + mainButtonFramePadding * 2 + ui.getWindowPadding().y * 2
+	local window_posx = ui.screenWidth/2 - ui.reticuleCircleRadius - window_width + 12 -- manual move a little closer to the center
+	local window_posy = ui.screenHeight - window_height
+	ui.setNextWindowPos(Vector2(window_posx, window_posy), "Always")
+	ui.window("ShipFunctions", {"NoTitleBar", "NoResize", "NoFocusOnAppearing", "NoBringToFrontOnFocus", "NoSavedSettings", "AlwaysAutoResize"},
```

Comment:
Rather than breaking the `function()` onto a new line, please move the window flags to file scope (using a `ui.WindowFlags` constructor) and bring the `function()` back onto this line. It's a minor thing, but it bugs me...

Reply:
Done!